pilot/pkg/proxy/envoy/watcher.go

```diff
@@ -39,23 +38,14 @@ type Watcher interface {
 	Run(context.Context)
 }
 
-// CertSource is file source for certificates
-type CertSource struct {
-	// Directory containing certificates
-	Directory string
-	// Files for certificates
-	Files []string
-}
-
 type watcher struct {
-	certs   []CertSource
+	certs   []string
```

Comment:
The watcher code now takes a list of absolute file paths to watch. This allows for maximum flexibility when cert file names and locations are dictated by outside forces.